Core/src/other/context/Context.java

```diff
@@ -1724,4 +1725,8 @@ public void storeCurrentData()
 		
 		trial.addUndoData(endData);
 	}
+
```

Comment:
I couldn't find which version this was using, and since the change was trivial decided to use this instead